sphinx/tutorial/rose/summary.rst

```diff
@@ -0,0 +1,199 @@
+            "rose-suite.conf" [fontsize="18", fontname="mono", fontcolor"black"]
+            "confinfo" [label="Defines Jinja2 variables for\nthesuite.rc and environment\nvariable for use throughout\nthe suite"]
+            "rose-suite.conf" -- "confinfo"
+
+            subgraph cluster_3 {
+                label = "Rosie Suite"
+                "rose-suite.info" [fontsize="18", fontname="mono", fontcolor"black"]
+                "infoinfo" [label="Contains basic information\nabout the suite used\nby Rosie for searching\nand version control purposes"]
+                "rose-suite.info" -- "infoinfo"
+            }
+        }
+    }
+
+Cylc suites can have Rose applications. These are stored in an ``app``
+directory and are configured using a :rose:file:`rose-app.conf` file.
+
+.. TODO - A file tree for an example Rose suite + a full file tree showing
+          all possible files and directories with descriptions, this will
+          require an extension and some CSS magic to ensure viable output
+          in HTML and PDF.
+
+
+Suite Commands
+--------------
+
+.. rubric:: We have learned the following Cylc commands:
+
+``cylc graph``
+   Draws the suite's :term:`graph`.
+``cylc get-config``
+   Processes the ``suite.rc`` file and prints it back out.
+``cylc validate``
+   Validate the Cylc ``suite.rc`` file to check for any obvious errors.
+``cylc run``
+   Runs a suite.
+``cylc stop``
+   Stops a suite:
+
+   ``--kill``
+      Killing all running / submitted tasks
+   ``--now --now``
+      Leaving all running / submitted tasks running.
+
+``cylc restart``
+   Starts a suite picking up where it left off.
+
+.. rubric:: We have learned the following Rose commands:
+
+:ref:`command-rose-app-run`
+   Runs a Rose application.
+:ref:`command-rose-task-run`
+   Used to run a Rose application from within a Cylc suite.
+:ref:`command-rose-suite-run`
+   Runs a Rose suite.
+:ref:`command-rose-suite-restart`
+   Runs a Rose suite picking up where it left off.
+
+The Cylc commands do not know about the :rose:file:`rose-suite.conf` file
+so for Rose suite configurations you will have to install the suite before
+using commands such as ``cylc graph`` e.g:
+
+.. code-block:: sub
+
+   rose suite-run -l  # install the suite on the local host only - don't run it.
+   cylc graph <suite> # run cylc graph using the installed version of the suite.
+
+
+Rose Utilities
+--------------
+
+Rose contains some utilities to make life easier:
+
+:ref:`command-rose-date`
+   A utility for parsing, manipulating and formatting date-times which is
+   useful for working with the Cylc :term:`cycle point`:
+
+   .. code-block:: console
+
+      $ rose date 2000 --offset '+P1Y1M1D'
+      2001-02-02T0000Z
+
+      $ rose date $CYLC_TASK_CYCLE_POINT --format 'The month is %B.'
+      The month is April.
+
+   See the :ref:`date-time tutorial <rose-tutorial-datetime-manipulation>`
+   for more information.
+
+:ref:`command-rose-host-select`
```

Comment:
I think we should get rid of this one.

Reply:
After https://github.com/cylc/cylc/issues/1885 and https://github.com/cylc/cylc/issues/2199?